Makefile

```diff
@@ -5,11 +5,11 @@
 #
 
 #
-# Copyright 2024 MNX Cloud, Inc.
+# Copyright 2025 MNX Cloud, Inc.
 #
 
 NAME = smartos-ui
-RUST_TOOLCHAIN = 1.75.0
+RUST_TOOLCHAIN = 1.84.1
```

Comment:
Does this bump the minimum PI that SmartOS UI needs?

Reply:
Oooh, I see the code to check for that is commented out, and dependent on manta...

```
        # # Get the minimium supported PI from the Manta object's metadata
        # minimium_pi=$("${CURL[@]}" -I -o /dev/null -w '%header{m-minimum-pi}' \
        #   "${URL_PREFIX}/smartos-ui-$version.tar.gz")

        # if [[ "${Live_Image:?}" < "$minimium_pi" ]]; then
        #       err "$version requires a platform image of $minimium_pi or newer"
        # fi
```
That starts at line 181 of uiadm.sh.